src/app/youtube/components/details/details.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Router } from '@angular/router';
+import { SearchResponse } from '../../models/search-response.model';
+import { YoutubeService } from '../../services/youtube.service';
+
+@Component({
+  selector: 'app-details',
+  templateUrl: './details.component.html',
+  styleUrls: ['./details.component.css']
+})
+export class DetailsComponent implements OnInit {
+
+  id: string = '';
+  videoThumbnail: string = '';
```

Comment:
Why you created SearchItem interface? 
please refactor this code remove all this fields.
create class `SearchItemModel `which implements `SearchItem`;
initialize `currentSearchItem: SearchItem = new  SearchItemModel ();`

Reply:
I refactored it, thank you